pymongo/vars.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License.  You
+# may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.  See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Internal helpers for CSOT."""
```

Comment:
I think `vars` is a confusing name for this module if it is meant for `csot` only.  Why not move this logic to `common` and expose a `_CSOT` module level variable there, or change this file name to `csot` or `csot_helpers`?